lib/ai/tools/reason-llm.ts

```
@@ -47,53 +55,82 @@ export async function executeReasonLLMTool({
+      (async () => {
+        const { fullStream } = streamText({
+          model,
+          system: systemPrompt,
+          messages: toVercelChatMessages(messages),
+          maxTokens: 8192,
+          abortSignal: abortSignal,
+          onFinish: async ({ finishReason }: { finishReason: string }) => {
+            if (supabase) {
+              await handleChatWithMetadata({
+                supabase,
+                chatMetadata: config.chatMetadata!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)